scripts/markdown_parse.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"regexp"
+	"strings"
+)
+
+const (
+	operationRegEx  = `(?s)<operation>.*?</operation>|(?s)<operation_json>.*?</operation>`
```

Comment:
Trying to have this be as close to xml/html as possible, this regexp would support weird input like: `<operation_json></operation>` 

Could you separate the regexps like: `(?s)<operation>.*?</operation>|(?s)<operation_json>.*?</operation_json>`

Reply:
Good point. Was trying to be more concise.